test/Microsoft.ML.Tests/Transformers/NormalizerTests.cs

```diff
@@ -217,5 +216,127 @@ public void TestWhiteningOldSavingAndLoading()
             }
             Done();
         }
+
+        [Fact]
+        public void LpNormWorkout()
+        {
+            var env = new ConsoleEnvironment(seed: 0);
```

Comment:
>ConsoleEnvironment [](start = 26, length = 18)

thought we ought to use Local

Reply:
actually, why don't you just use the provided `ML` as an env?

---
In reply to: [232162522](https://github.com/dotnet/machinelearning/pull/1582#discussion_r232162522) [](ancestors = 232162522)

Reply:
I mean, `this.ML`

---
In reply to: [232436854](https://github.com/dotnet/machinelearning/pull/1582#discussion_r232436854) [](ancestors = 232436854,232162522)